### DISCUSSION TITLE
Advertise my projects that uses on JAX

### DISCUSSION THREAD

**Discussion by ntlm1686** [jax-ml/jax#14114] [Q&A] [answered]
Hi,

Is there a place for me to advertise my projects which use JAX in the JAX's community?

**Comment by adam-hartshorne** [comment, marked as answer]
https://github.com/n2cholas/awesome-jax

**Reply by ntlm1686** [reply]
Thanks

**Comment by soraros** [comment]
There is also the "show and tell" category in GitHub discussion.
